http/oidc/src/test/java/org/wildfly/security/http/oidc/BearerTest.java

```diff
@@ -0,0 +1,545 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2022 Red Hat, Inc., and individual contributors
```

Comment:
2023?

Reply:
This file is part of the backported commits.